app/views/unwithdraw/_confirm.html.erb

```diff
@@ -0,0 +1,20 @@
+<% confirmation = capture do %>
+<%= form_tag(unwithdraw_path(@document), method: :post) do %>
+
+  <p><%= t("documents.show.unwithdraw.title") %></p>
+
+  <%= render "govuk_publishing_components/components/button", {
+    text: t("documents.show.unwithdraw.confirm"),
+    destructive: true
+  } %>
+
+  <div style="display: inline-block; padding-left: 15px; padding-top: 8px">
```

Comment:
I think you're already aware of this, but this is something that we already changed on discard draft and this should be updated for it.